src/app/visual/repeater/repeater-visual.component.ts

```diff
@@ -91,6 +106,16 @@ export class RepeaterVisualComponent {
     this.items.push(newItem);
   }
 
+  public getSelectedItems(): string[] {
+    const ids: string[] = [];
+    this.itemsForSelectableRepeater.forEach(item => {
+      if (item.isSelected) {
+        ids.push(item.id.toString());
+      }
+    });
+    return ids;
```

Comment:
Not a huge deal, but this logic could be simplified:

```
const ids: string[] = this.itemsForSelectableRepeater
  .filter(item => item.isSelected)
  .map(item => item.id.toString());

return ids;
```

Reply:
I like it! Done.